substrate/frame/safe-mode/src/tests.rs

```diff
@@ -22,8 +22,7 @@
 use super::*;
 use crate::mock::{RuntimeCall, *};
 
-use frame_support::{assert_err, assert_noop, assert_ok, hypothetically_ok, traits::Currency};
-use sp_runtime::traits::Dispatchable;
+use frame::{deps::frame_support::hypothetically_ok, testing_prelude::*, traits::Currency};
```

Comment:
`hypothetically_ok` can be reexported in `testing_prelude`.

Reply:
the reason i didnt export it is because its only used in 2 crates, is it enough to justify putting it in the prelude ?